src/FeedbackHelpful.ce.vue

```diff
@@ -0,0 +1,291 @@
+            {{ t('no') }}
+          </radio-item>
+          <span class="label label-danger">{{ errors.isHelpful }}</span>
+          <fieldset id="feedbackNoFieldset" v-if="getIsHelpful == Answer.NO">
+            <legend class="required">
+              <span class="field-name">{{ t("why") }}? {{ t("selectOptions") }}</span><strong class="required"> ({{ t("required") }})</strong>{{ t('punctuation.colon') }}
+            </legend>
+            <!-- vee-validate parameters only have to be set on one element with the same name -->
+            <checkbox-item name="reasons" id="infoHardToUnderstand" :value="Reasons.INFO_HARD_TO_UNDERSTAND"
+              :is-reset="isResetReasons" @reset-completed="handleResetReasonsCompleted" @change="handleReasonsChanged()"
+              :rules="{ required: true }">
+              {{ t("form.infoHardToUnderstand") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="infoWrong" :value="Reasons.INFO_WRONG" @change="handleReasonsChanged()">
+              {{ t("form.infoWrong") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="helpfulInfoOutdated" :value="Reasons.INFO_OUTDATED"
+              @change="handleReasonsChanged()">
+              {{ t("form.infoOutdated") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="infoNotFound" :value="Reasons.INFO_NOT_FOUND"
+              @change="handleReasonsChanged()">
+              {{ t("form.infoNotFound") }}
+            </checkbox-item>
+            <checkbox-item name="reasons" id="helpfulOther" :value="Reasons.OTHER" @change="handleReasonOtherChanged()">
+              {{ t("form.other") }}
+              <div class="form-group" v-if="isReasonOtherSelected">
+                <textarea-input name="comment" id="commentOtherNotHelpful" maxLength.number="750"
+                  :rules="{ required: true, max: 750 }">
+                  {{ t("commentSpecifyDetails") }}
+                </textarea-input>
+                <span class="label label-danger">{{ errors.comment }}</span>
+              </div>
+            </checkbox-item>
+            <span class="label label-danger">{{ errors.reasons }}</span>
+          </fieldset>
+        </fieldset>
+        <!-- Limitation: Can't pass i18n from a custom component to a vue component -->
+        <div>
+          <div class="alert alert-info">
+            <h2 class="h5">{{ t('submissionInformation.note') }}</h2>
+            <ul>
+              <li><small><strong>{{ t('contact.noReply') }}.</strong> {{ t('contact.forEnquiries') }} <a
+                    :href="`${websiteBasePath}${t('contact.opcContactLink')}`">{{ t('contact.contactOpc')
+}}</a>.</small>
+              </li>
+              <li><small><strong>{{ t('submissionInformation.noPersonalInfo') }},</strong> {{
+    t('submissionInformation.personalInfoItems')
+}}.</small></li>
+              <li><small><strong>{{ t('submissionInformation.forMoreInfo') }}</strong>, {{
```

Comment:
I wonder if https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage would be cleaner to generate the `li`.

Reply:
If you decide to move forward with this, there's a few other places that can use the same fix.

Reply:
It doesn't seem to be compatible with web components at the moment.